shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -877,13 +913,45 @@ SemanticsObject hitTest(float[] point) {
         boolean isFocusable() {
             int scrollableActions = Action.SCROLL_RIGHT.value | Action.SCROLL_LEFT.value
                     | Action.SCROLL_UP.value | Action.SCROLL_DOWN.value;
+            if (hasFlag(Flag.IS_EDGE) && value == null) {
```

Comment:
Do we enforce on the framework side that no other relevant information get merged into the IS_EDGE node?

Reply:
Not yet, but I can make sure the framework created semantics nodes have explicitChildNodes set